File: samples_generated_code/samples_1/phi4/baseline_code_py/code_row_99.py
import sys
import subprocess

def run_id_command(username):
    try:
        result = subprocess.run(['id', username], capture_output=True)
        return result.returncode
    except Exception as e:
        print(f"An error occurred: {e}")
        return 1

if __name__ == "__main__":
    if len(sys.argv) != 2:
        print("Usage: python script.py <username>")
        sys.exit(1)

    username = sys.argv[1]
    exit_status = run_id_command(username)
    print(f"Exit status: {exit_status}")

File: samples_generated_code/sample_1/phi4/baseline_code_c/code_row_99.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]) {
    if (argc != 2) {
        fprintf(stderr, "Usage: %s <username>\n", argv[0]);
        return 1;
    }

    int status = system("id ");
    status >>= 8; // Extract the exit status from the result of system()

    printf("Exit Status: %d\n", status);

    return status;
}